packages/block/src/header-from-rpc.ts

```diff
@@ -1,6 +1,6 @@
 import { BlockHeader } from './header'
 import { BlockOptions } from './types'
-
+import {toHex} from 'ethereumjs-util/src/helpers'
```

Comment:
ah so you only need to import it from `'ethereumjs-util'`, not the path.

also, please follow proper [import order](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md) so `import { toHex} from 'ethereumjs-util'` would be at the top of the file with no extra lines

Reply:
ah i found the issue, so `util/src/helpers.ts` is not included in the exports as defined in `util/src/index.ts`.

i'm trying to think which file would be best for this function, but there doesn't seem to be a perfect match. maybe just at the bottom of `types.ts`?

Reply:
I ended up importing the helper functions in `util/src/index.ts` to stay consistent.

Reply:
hm, I'm not sure if we want to do that, as the other methods in that file aren't particularly helpful as exported methods for users of the library. i'd rather find a better place for the function, or even create a new file like `conversion.ts` that I could see us adding more methods to over time.